pkg/services/librarypanels/librarypanels.go

```diff
@@ -171,3 +181,28 @@ func importLibraryPanelsRecursively(c context.Context, service libraryelements.S
 
 	return nil
 }
+
+// CountInFolder is a handler for retrieving the number of library panels contained
+// within a given folder and for a specific organisation.
+func (lps LibraryPanelService) CountInFolder(ctx context.Context, orgID int64, folderUID string, u *user.SignedInUser) (int64, error) {
+	var count int64
+	return count, lps.SQLStore.WithDbSession(ctx, func(sess *db.Session) error {
+		folder, err := lps.FolderService.Get(ctx, &folder.GetFolderQuery{UID: &folderUID, OrgID: orgID, SignedInUser: u})
+		if err != nil {
+			return err
+		}
+
+		q := sess.Table("library_element").Where("org_id = ?", u.OrgID).
```

Comment:
I'm a little bit concerned that the `UQE_library_element_org_id_folder_id_name_kind` index on 
`org_id`,`folder_id`,`name`,`kind` may only partially used (because of the missing `name` col)
I would worth checking its performance (in a follow up) for all the different backends.

Reply:
I wonder also if we start counting also the library variables except from the library panels

Reply:
I had asked about library variables earlier: [Slack](https://raintank-corp.slack.com/archives/C045Y3PLNET/p1684956067292539?thread_ts=1684937793.375929&cid=C045Y3PLNET). Seems it's not something we need to take into account at this time.

Reply:
> I would worth checking its performance (in a follow up) for all the different backends.

👍 FYI https://github.com/grafana/grafana/issues/72300